src/VecSim/vec_sim.h

```diff
@@ -49,12 +49,21 @@ int VecSimIndex_DeleteVector(VecSimIndex *index, size_t id);
  * metric.
  * @param id the id of the vector in the index.
  * @param blob binary representation of the second vector. Blob size should match the index data
- * type and dimension.
+ * type and dimension, and pre-normalized if needed.
```

Comment:
`and pre-normalized if needed.`
This should be more emphasised in my opinion, as it is not trivial...